testing/crypto++/APKBUILD

```
@@ -4,33 +4,36 @@ pkgname=crypto++
 _pkgname=cryptopp
 pkgver=5.6.5
 _pkgver=${pkgver//./}
-pkgrel=1
+pkgrel=2
 pkgdesc="A free C++ class library of cryptographic schemes"
 url="https://www.cryptopp.com/"
 arch="all"
 license="BSL-1.0"
-depends=""
 depends_dev="$pkgname"
-makedepends=""
 subpackages="$pkgname-dev"
-source="https://www.cryptopp.com/${_pkgname}${_pkgver}.zip"
+source="https://www.cryptopp.com/${_pkgname}${_pkgver}.zip
+	crypto++.pc
+	"
 
 builddir="$srcdir"
 
 build() {
-	cd "$builddir"
-	sed -i -e 's/^CXXFLAGS/#CXXFLAGS/' GNUmakefile || return 1
+	sed -i -e 's/^CXXFLAGS/#CXXFLAGS/' GNUmakefile
 	export CXXFLAGS="${CXXFLAGS} -DNDEBUG -fPIC"
-	make -f GNUmakefile || return 1
-	make libcryptopp.so || return 1
+	make dynamic cryptest.exe
+}
+
+check() {
+	make test
 }
 
 package() {
-	cd "$builddir"
-	install -Dm644 libcryptopp.so "$pkgdir"/usr/lib/libcryptopp.so || return 1
-	install -Dm644 License.txt "$pkgdir"/usr/share/licenses/$pkgname/LICENSE || return 1
-	mkdir -p "$pkgdir"/usr/include/cryptopp/ || return 1
-	install -m644 *.h "$pkgdir"/usr/include/cryptopp/ || return 1
+	make install DESTDIR="$pkgdir" PREFIX=/usr
+	rm -rf "$pkgdir"/usr/bin "$pkgdir"/usr/share/cryptopp
+
+	install -Dm644 "$srcdir"/crypto++.pc "$pkgdir"/usr/lib/pkgconfig/crypto++.pc
+	sed -e "s,@@VERSION@@,${version},g" -i "$pkgdir"/usr/lib/pkgconfig/libcrypto++.pc
```

```suggestion
	sed -e 's,@@VERSION@@,${version},g' -i "$pkgdir"/usr/lib/pkgconfig/libcrypto++.pc
```

version is not set earlier, so I guess you don't want to expand to empty here.